aws/eks/helm.tf

```diff
@@ -157,6 +157,13 @@ resource "helm_release" "kubernetes_external_secrets" {
   }
 }
 
+module "configmapsecrets" {
```

Comment:
so this would setup configmap secrets for all EKS clusters we create using this eks terraform module - am I reading this correctly?

Reply:
(with the configmap secrets boolean trigger guiding that; i see the default is to not install)